msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -51,6 +51,36 @@ Even if you intend your app to only run in Teams (and not Microsoft 365 app and
 
 Once you're able, the next step is to [update existing application code](#2-update-teamsjs-references) with the changes described in this article. In the meantime, the v.1 to v.2 API translation layer provides backwards compatibility, ensuring your existing Teams app continues to work in TeamsJS version 2.0.
 
+To implement Teams app-specific logic, include the following code snippet to your app that initializes the TeamsSDK and runs in Teams platform exclusively.
+
+```js
+import 'https://res.cdn.office.net/teams-js/2.0.0/js/MicrosoftTeams.min.js';
+
+// Ensure that the Teans SDK is initialized once no matter how often this is called
+let teamsInitPromise;
+export function ensureTeamsSdkInitialized(){
+    if (!teamsInitPromise) {
+        teamsInitPromise = microsoftTeams.app.initialize();
+    }
+    return teamsInitPromise;
+}
+
+// Function returns a promise which resolves to true if we're running in Teams
+export async function inTeams(){
+  try {
+    await ensureTeamsSdkInialized();
+    const context = await microsoftTeams.app.getContext();
+    return (context.app.host.name === microsoftTeams.HostName.teams);
+  }
+  catch (e) {
+    console.log(`${e} from Teams SDK, may be running outside of Teams`);
+    return false;
+  }
+}                                                                                                                                
+```
+
+You must wait for the [app initialization](/javascript/api/@microsoft/teams-js/app#@microsoft-teams-js-app-isinitialized) to complete before proceeding with the function call in order for the app to function correctly. Any program logic designed exclusively for Teams might not function correctly on other Microsoft 365 platforms. To ensure the smooth operation of your app across the Microsoft 365 ecosystem, make provisions for the logic handling of other Microsoft 365 platforms.
```

Comment:
Instead of "platforms", let's call them _applications_. (More specific and accurate.)

Reply:
Updated.